jlm/mlir/backend/JlmToMlirConverter.cpp

```diff
@@ -62,17 +63,13 @@ JlmToMlirConverter::Print(::mlir::rvsdg::OmegaNode & omega, const util::filepath
 ::mlir::rvsdg::OmegaNode
 JlmToMlirConverter::ConvertModule(const llvm::RvsdgModule & rvsdgModule)
 {
-  return ConvertOmega(rvsdgModule.Rvsdg());
```

Comment:
Need to get https://github.com/phate/jlm/pull/888 merged before merging this one